packages/ra-ui-materialui/src/layout/RecordTitle.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Title from './Title';
+import TitleDeprecated from './TitleDeprecated';
 
+/**
+ * @deprecated Use TitleForRecord instead
+ */
 const RecordTitle = ({ defaultTitle, record, title }) =>
```

Comment:
Aren't we missing a console warning here?

Reply:
since all occurrences of `RecordTitle` were inside a `Header`, and we already warn about it, that would trigger two warnings in a row... I think one is enough.